roles/pulp_common/tasks/install_packages.yml

```diff
@@ -32,6 +32,9 @@
             src: /etc/dnf/dnf.conf
             dest: "{{ __dnf_conf.path }}"
             remote_src: true
+            mode: 0644
+            owner: root
+            group: root
```

Comment:
didn't we recently have a PR to avoid root as the owner?

Reply:
It was become_user
https://github.com/pulp/pulp_installer/pull/843

Reply:
Yeah, that pertains to the root user, but is very different.

We just shouldn't do `become_user: root`